src/components/Auth/Login.js

```diff
@@ -1,6 +1,53 @@
+import axios from "axios";
 import React from "react";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuthContex } from "../../contex/auth-contex";
+
 import "./auth.css";
 const Login = () => {
+  const navigateHome = useNavigate();
+  const { authDispatch } = useAuthContex();
+  const [userDetail, setUserDetail] = useState({
+    email: "",
+    password: "",
+  });
+  const testUser = {
+    email: "adarshbalak@gmail.com",
+    password: "adarshBalaki123",
+  };
+  const setUserDetailHandler = (e) => {
+    let name = e.target.name;
+    let value = e.target.value;
```

Comment:
- You can write both of these lines in one line, like:- `const {name,target} = e.target`.
- Also try to use `const` as most of the places, use `let` only when you are changing that variable's value inside that functions. Since it is not happening here, you can remove let and use const over it.